kokoro/release/python/linux/build_artifacts.sh

```diff
@@ -25,7 +25,13 @@ rm -rf multibuild/
 mkdir artifacts
 export ARTIFACT_DIR=$(pwd)/artifacts
 
+# Pin multibuild script to a version just before the default
+# manylinux image has switched from manylinux1 to manylinux2014.
+# Also, pinning version avoid potentially unwanted future changes from
+# silently creeping in.
+# See https://github.com/protocolbuffers/protobuf/issues/9180
 git clone https://github.com/matthew-brett/multibuild.git
+(cd multibuild; git checkout 13a01725b0f0aa551ab34aa2311cdc1c77be4337)
```

Comment:
For future reference, the manylinux version can be set by setting 
`MB_ML_VERSION=1` when invoking the `build_wheel`.  But in our case it should only be done for x86_64 artifacts (aarch64 is only supported starting from manylinux2014)
https://github.com/matthew-brett/multibuild/blob/b89bb903e94308be79abefa4f436bf123ebb1313/travis_linux_steps.sh#L71